modules/examiner/README.md

```diff
@@ -0,0 +1,82 @@
+# Examiner
+
+## Purpose
+
+The Examiner module displays the list of examiners in the study and allows 
+for additions and removal of certifications for each examiner listed.
+
+## Intended Users
+
+The primary type of users is:
+1. Site coordinators adding examiners and adding certifications to examiners
+
+
+## Scope
+
+This module displays information about examiners as well as certifications and 
+sites to which the examiners are affiliated. It also allows to add examiners 
+to sites, to give or remove certifications for instruments and set them as 
+radiologists. The Examiner architecture is completely separate from the User 
+architecture intentionally; examiners may or may not be users.
+
+NOT in scope:
+
+Management of the certification instruments and of the sites is external to 
+this module.
+
+## Permissions
+
+The Examiner module uses the following permissions. Any one of them 
+is sufficient to have access to the module.
+
+examiner_view
+ - This permission gives the user viewing and editing access to all 
```

Comment:
is the "and editing" a typo or is that what the view permission does?

Reply:
nop
hahaha